src/main/resources/io/jenkins/plugins/coverage/adapter/CoverageReportAdapterDescriptor/config.jelly

```diff
@@ -10,4 +10,49 @@
             </tr>
         </table>
     </f:entry>
+
+    <f:advanced>
```

Comment:
Add reference to the source implementation. License header may also need to be adjusted a bit